adoc/Release-Notes.adoc

```diff
@@ -31,6 +31,124 @@ endif::[]
 [id='sec.major-change']
 == Major Changes
 
+[id='sec.1_3_1']
+=== Release 1.3.1, February 2019
+
+[id='sec.1_3_1.new']
+==== What Is New?
+
+* {scf} has been updated to version 2.15.2:
+** Default PodSecurityPolicies (PSPs) come with the helm charts
+** cflinuxfs3 now available as a stack
+** Added nginx buildpack
+** Support added for placement zones & isolation segments
+* The Stratos UI has been updated to version 2.3:
+** Support for extensions
+** For a full list of features and fixes see https://github.com/SUSE/stratos/releases/tag/2.3.0.
+
+For information about deploying and administering {product}, see the product manuals at
+{doc-url}.
+
+
+[id='sec.1_3_1.feature']
+==== Features and Fixes
+
+* App-autoscaler no longer dependent on hairpin
+* Credhub on Azure is now supported
+* Corrected service name to work with `syslog` drains
+* Certificates rely on correct UAA FQDN
```

Comment:
UAA already exists throughout the release notes as its initials since that's the upstream convention. I'll reword this to "Certificates rely on correct FQDN for UAA" for now and discuss with product (who still has to review)